tests/test_symmetric_matrices.py

```diff
@@ -0,0 +1,32 @@
+"""Unit tests for the vector space of symmetric matrices."""
+
+import warnings
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.symmetric_matrices import SymmetricMatrices
+
+
+class TestSymmetricMatricesMethods(geomstats.tests.TestCase):
+    """Test of SymmetricMatrices methods."""
+
+    def setUp(self):
+        """Set up the test."""
+        warnings.simplefilter('ignore', category=ImportWarning)
+
+        gs.random.seed(1234)
+
+        self.n = 3
+
+    def test_belongs(self):
+        """Test of belongs method."""
+        Sym_n = SymmetricMatrices(self.n)
```

Comment:
Ok since the coding style in unit tests is less strict, but remember than python (pep8) convention is to have all variable names in lower-cases: so this should be `sym_n`. It should only have an uppercase if you were using the whole class, eg:
Sym = SymmetricMatrices.